src/vm/internal/compiler/lex.cc

```diff
@@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
   }
 }
 
+void init_include_path() {
+  svalue_t *ret;
+  array_t *arr;
+  const char *elem;
```

Comment:
I had some problems withnmy net connection :(
And now somehow your complain concerning thís patch got missing…
Could you please repost what I shall fix?